php/Dockerfile-debian

```diff
@@ -2,7 +2,7 @@
 # ==============================================
 
 ARG PHP_BASE_IMAGE_VERSION
-FROM php:${PHP_BASE_IMAGE_VERSION}
+FROM yiisoftware/yii-php:${PHP_BASE_IMAGE_VERSION}
```

Comment:
This repo is for building `yiisoftware/yii-php`, this should be reverted.

Rest is fine.